pkg/cmd/generate/cluster.go

```diff
@@ -16,7 +15,7 @@ type clusterContext struct {
 func ensureServiceAccounts(ctx *clusterContext, objsCache objectsCache) error {
 	ctx.Printlnf("-> Ensuring ServiceAccounts and its RoleBindings...")
 	for _, sa := range ctx.kubeSawAdmins.ServiceAccounts {
-		if ctx.specificKMemberName != "" && slices.Contains(sa.Selector.SkipMembers, ctx.specificKMemberName) {
+		if sa.Selector.ShouldBeSkippedForMember(ctx.specificKMemberName) {
```

Comment:
should we keep the empty name check ?

Reply:
it's already part of `ShouldBeSkippedForMember` method logic